nibabel/info.py

```diff
@@ -209,4 +209,5 @@ def cmp_pkg_version(version_str, pkg_version_str=__version__):
 ISRELEASE = _version_extra == ''
 VERSION = __version__
 PROVIDES = ["nibabel", 'nisext']
-REQUIRES = ["numpy (>=%s)" % NUMPY_MIN_VERSION]
+REQUIRES = ["numpy (>=%s)" % NUMPY_MIN_VERSION,
```

Comment:
We still need to use the `bz2file.BZ2File` object instead of the the `bz2.BZ2File` object, in `openers`, for Python 2.  Maybe:

```
import sys
if sys.version_info[0] < 3:
    from bz2file import BZ2File
else:
    from bz2 import BZ2File
```

and so on, maybe importing from here for other uses of `BZ2File`.  I can't think immediately of a test for saving memory here, can you?

Reply:
As in to avoid an early import? `pkgutil.find_loader()` is a Python 2/3-safe way to see if a module can be imported without actually importing.

Reply:
No, sorry, the `bz2file` package provides an alternative `BZ2File` implementation, it does not replace the implementation in the `bz2` package.   If you check, I think you'll find that `bz2.BZ2File` still doesn't have `readinto` after you've installed `bz2file`, but `bz2file.BZ2File` does have `readinto` (on Python 2.7).

Reply:
Ah, reparsed. Yeah, I still need to make that change. On the whole, this PR just isn't ready. To get tests working again, I still need to figure out why we can't use version selectors in `install_requires`. I suspect it's a `distutils` or `setuptools` problem.